test/difftree.sh

```diff
@@ -46,7 +46,7 @@ do
 		;;
 	esac
 	# skip objects that are the linked product of more than one object file
-	[[ $(eu-readelf -s $i | grep FILE | wc -l) -ne 1 ]] && continue
+	[[ $(readelf -s $i | awk -F " " '{print $4}' | grep FILE | wc -l) -ne 1 ]] && continue
```

Comment:
This could be simplified a little bit by incorporating the filtering and counting into the awk script:
```
readelf -s $(find . -name vmlinux) | awk '$4=="FILE" {n++} END {print n}'
```

Reply:
> This could be simplified a little bit by incorporating the filtering and counting into the awk script:
> 
> ```
> readelf -s $(find . -name vmlinux) | awk '$4=="FILE" {n++} END {print n}'
> ```

yet, but I don't think we can use "$(find . -name vmlinux)" replace the "$i".